extensions/imdb/README.md

```diff
@@ -0,0 +1,44 @@
+<p align="center">
+  <img src="assets/icon.png" height="128">
+  <h1 align="center">IMDb Search</h1>
+</p>
+
+A [Raycast](https://raycast.com/) extenstion that lets you easily open your favorite film or tv series on [IMDb](https://imdb.com/).
+
+This extension is made by [Ryan Voevodin](https://twitter.com/RyanVoevodin) and uses [The Open Movie Database API](https://www.omdbapi.com/) (OMDb) by Brian Fritz.
+
+![Example of IMDb Search Extension on Raycast](https://ryan-exe.s3.ap-southeast-2.amazonaws.com/imdb-raycast/showcase.png)
+
+## Getting your API Token
+
+1. Visit the [OMDb website](https://www.omdbapi.com/apikey.aspx).
+2. Sign up to the `Free (1000 Daily Limit)` option with your email address.
+3. Check your inbox and activate your API key from OMDb.
+   > Note: You may need to change the activation link to `https://`
+4. Copy your key e.g. `ab12c345` onto the IMDb Search preference pane on Raycast (see image below).
+5. You're all ready to go, start searching.
+
+![Enter your API on Preference Pane](https://ryan-exe.s3.ap-southeast-2.amazonaws.com/imdb-raycast/enterAPI.png)
+
+## Troubleshooting
+
+### 🔺 401 [Unauthorized]
+
+The API key you've provided either hasn't been activated or is incorrect.
+
+1. Open the confirmation email from OMDb.
+2. Activate the API key once more.
+3. On Raycast open the Preferences pane (Search "Extensions Raycast Preferences").
+4. Select the `Search Film or TV Series` command from the list.
```

Comment:
```suggestion
4. Select the `Search Film and TV Series` command from the list.
```

Since it's not possible to select search for one alone, it should be “and”